one-ui/ui/src/util/user-context.tsx

```diff
@@ -30,7 +34,8 @@ const defaultUserData = {
   },
   tableView: true,
   pageRoute: '/view',
-  maxSessionTime: 300
+  maxSessionTime: 300,
```

Comment:
Max session time is how long the session is in seconds. Default is 300 = 5 mins. There's an endpoint call where I get the session time for the server. Changing it to 60 won't do anything, because the maxSession time gets overwritten with the server parameter.

Reply:
If that value is getting overwritten by the value received from the server, don't worry about storing a configurable version as I suggested.